keyboards/sol/keymaps/danielhklein/keymap.c

```diff
@@ -3,8 +3,8 @@
 #include "lufa.h"
 #include "split_util.h"
 #endif
-#ifdef SSD1306OLED
-  #include "common/ssd1306.h"
+#ifdef OLED_DRIVER_ENABLE
```

Comment:
Ugh, no.

This should be in `quantum.h`, so that users do not manually have to add this.

Reply:
Fixed